types/node/os.d.ts

```diff
@@ -28,6 +28,7 @@ declare module 'os' {
     }
     interface NetworkInterfaceInfoIPv4 extends NetworkInterfaceBase {
         family: 'IPv4';
+        scopeid: undefined;
```

Comment:
```suggestion
        scopeid?: undefined;
```
there is no single convention, that one would be aligned with most of the other entires and with support for strict optional property types